logbook-spring/src/main/java/org/zalando/logbook/spring/BufferingClientHttpResponseWrapper.java

```diff
@@ -48,7 +48,9 @@ public void close() {
         } catch (IOException e){
             throw new RuntimeException(e);
         }
-        delegate.close();
+        finally {
+            delegate.close();
```

Comment:
That's a really nice catch, @bobross-pg ! Would you also add a test for this scenario? So that we don't hit regression in future.